draft-hale-pquip-hybrid-signature-spectrums.md

```diff
@@ -210,21 +212,22 @@ certificate' as defined in [RFC4949].
 
 - Hybrid signature scheme: Following [I-D.ietf-pquip-pqt-hybrid-terminology], we
   define a hybrid signature scheme to be "a multi-algorithm digital signature
-  scheme made up of two or more component digital signature algorithms ...". We
+  scheme made up of two or more component digital signature algorithms ...". While it often makes sense for security purposes to 
   require that the security of the component schemes is based on the hardness of
-  different cryptographic assumptions. In contrast to
+  different cryptographic assumptions, in other cases hybrid schemes might be motived, e.g., by interoperatbility of variants on the same scheme and as such both component schemes are based on the same hardness assumption. We allow this explicitely. This means in particular that in contrast to
   [I-D.ietf-pquip-pqt-hybrid-terminology], we will use the more general term
   'hybrid signature scheme' instead of requiring one post-quantum and one
   traditional algorithm (i.e., PQ/T hybrid signature schemes) to allow also the
-  combination of several post-quantum algorithms. The term 'composite' scheme is
-  often used as a synonym for 'hybrid scheme'. This is different from
-  [I-D.ietf-pquip-pqt-hybrid-terminology] where the term is used at the protocol
-  level.
+  combination of several post-quantum algorithms. The term 'composite scheme' is
+  sometimes used as a synonym for 'hybrid scheme'. This is different from
+  [I-D.ietf-pquip-pqt-hybrid-terminology] where the term is used as a specific instantiation of hybrid schemes such that "where
+   multiple cryptographic algorithms are combined to form a single key
+   or signature such that they can be treated as a single atomic object
+   at the protocol level." To avoid confusing we will avoid the term 'composite scheme'. 
```

Comment:
```suggestion
  combination of several post-quantum algorithms. The term 'composite scheme' is
  sometimes used as a synonym for 'hybrid scheme'. This is different from
  [I-D.ietf-pquip-pqt-hybrid-terminology] where the term is used as a specific
  instantiation of hybrid schemes such that "where multiple cryptographic
  algorithms are combined to form a single key or signature such that they can
  be treated as a single atomic object at the protocol level." To avoid confusing
  we will avoid the term 'composite scheme'. 
```